WORKSPACE

```diff
@@ -235,3 +202,28 @@ http_archive(
     strip_prefix = "libevent-release-2.1.8-stable",
     build_file = "@//third_party/libevent:BUILD",
 )
+
+##################### OPEN VINO ######################
+# OPENVINO DEFINITION FOR BUILDING FROM BINARY RELEASE: ##########################
+new_local_repository(
+    name = "openvino",
+    build_file_content = """
+cc_library(
+    name = "openvino",
+    srcs = glob([
+        "inference_engine/lib/intel64/libinference_engine_legacy.so",
+        "inference_engine/lib/intel64/libinference_engine.so",
+        "inference_engine/lib/intel64/libinference_engine_c_api.so",
+        "ngraph/lib/libngraph.so"
+    ]),
+    hdrs = glob([
+        "inference_engine/include/**/*.*"
+    ]),
```

Comment:
Can we move the build_file_content to third_party/openvino/ and use build_file instead (similarly to awssdk above) ?